srcCpp/perftest_publisher.cxx

```diff
@@ -1213,7 +1216,7 @@ int perftest_cpp::Subscriber()
     }
 
     PerftestClock::milliSleep(2000);
-
+    _printer.print_finish_output();
```

Comment:
Small comment explaining what this does.

Reply:
ok